src/i18n/strings/fr.json

```diff
@@ -2633,7 +2633,9 @@
             "show_checklist_shortcuts": "Afficher le raccourci vers la liste de vérification de bienvenue au-dessus de la liste des salons",
             "show_polls_button": "Afficher le bouton des sondages",
             "surround_text": "Entourer le texte sélectionné lors de la saisie de certains caractères",
-            "time_heading": "Affichage de l’heure"
```

Comment:
Please don't edit the i18n files except `en_EN.json`, the translation are handled by element and the community  https://github.com/element-hq/element-web/blob/develop/docs/translating.md